pkg/logs/status/status.go

```diff
@@ -113,6 +115,20 @@ func Get() Status {
 	}
 }
 
+// AddGlobalWarning create a warning
+func AddGlobalWarning(key string, warning string) {
```

Comment:
It's idempotent because of the map implementation. There is a test case for this here https://github.com/DataDog/datadog-agent/commit/362c782af3b19d3e2d3031c1a38a3ced654ee4ac#diff-8dcf77ed16f390e66721f32cbeb223daR46

Reply:
👍